tests/components/updater/test_init.py

```diff
@@ -51,6 +51,10 @@ async def test_new_version_shows_entity_true(hass, mock_get_newest_version):
         hass.states.get("binary_sensor.updater").attributes["release_notes"]
         == RELEASE_NOTES
     )
+    assert (
+        hass.states.get("binary_sensor.updater").attributes["current_version"]
```

Comment:
NIT, but maybe we could change this to:
```
state = hass.states.get("binary_sensor.updater")
assert state.state == STATE_ON
assert state.attributes["newest_version"] == NEW_VERSION
assert state.attributes["release_notes"] == RELEASE_NOTES
assert state.attributes["current_version"] == MOCK_VERSION
```
This way we avoid multiple hass.states.get() function calls.

Reply:
That's an improvement, a separate PR cleaning that up would be welcome 👍

Reply:
> That's an improvement, a separate PR cleaning that up would be welcome 👍

Okay, I thought it could be changed in this PR since we're only minimally changing the tests...

Reply:
It's generally better to separate code improvements from new features.
In this case it doesn't really matter, but we should not block merging this PR because the existing tests could be improved.

Reply:
> It's generally better to separate code improvements from new features. In this case it doesn't really matter, but we should not block merging this PR because the existing tests could be improved.

Yeah, I also think it's better to separate code improvements from new features but, in my opinion, creating a separate PR for that has a downside: it's another PR to review which will diminish time from reviewing other more important PRs and the project is already saturated to the point that new integrations generally take months to be accepted (even if they require minimal changes to be merged.)
So in the end, only integrations created by the official project members are developed and integrated in an acceptable period of time.
But that's my personal opinion ¯\_(ツ)_/¯